libs/search-widget/src/old-components/viewer/Metadata.svelte

```diff
@@ -164,29 +159,30 @@
       {/if}
 
       {#each $files || [] as file}
-        <a class="download" href={file}>
-          <img src={`${getCDN()}icons/source.svg`} alt="icon"/>
+        <a class="download" href={file} target="_blank">
+          <img src={`${getCDN()}icons/source.svg`} alt="icon" />
           <div>{$_('resource.source')}</div>
         </a>
       {/each}
 
       {#each $links as link}
         <a class="download" href={link} rel="noopener noreferrer" target="_blank">
-          <img src={`${getCDN()}icons/source.svg`} alt="icon"/>
+          <img src={`${getCDN()}icons/source.svg`} alt="icon" />
           <div>{$_('resource.source')}</div>
         </a>
       {/each}
     </div>
   {/if}
 
-  <ConfirmDialog show={showSafariModal}
-                 buttons={[{label: 'Ok', action: 'confirm'}]}
-                 closeable
-                 on:cancel={() => showSafariModal = false}
-                 on:confirm={() => showSafariModal = false}>
+  <ConfirmDialog
+    show={showSafariModal}
+    buttons={[{ label: 'Ok', action: 'confirm' }]}
+    closeable
+    on:cancel={() => (showSafariModal = false)}
+    on:confirm={() => (showSafariModal = false)}
+  >
```

Comment:
still those reformatting hitting on us :( 
maybe we should proritize the linter ticket you created some time ago?

Reply:
right